docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -84,7 +91,10 @@ def events(context: AssetExecutionContext):
     overwrite_data_in_datetime_range(start_datetime, end_datetime, output_data)
 ```
 
-If you are using an I/O manager to handle saving and loading your data, you'll need to ensure the I/O manager is also using these methods. If you're using any of the built-in database I/O managers, like [Snowflake](/integrations/snowflake), [BigQuery](/integrations/bigquery), or [DuckDB](/\_apidocs/libraries/dagster-duckdb), you'll have this out-of-the-box. **Note**: This doesn't apply to file system I/O managers.
+</TabItem>
+<TabItem name="With I/O managers">
+
+#### With I/O managers
```

Comment:
I think including the asset in the I/O manager example would also be helpful since now the two examples are comparing the different approaches, vs being more linear 

```python
@asset(
    partitions_def=DailyPartitionsDefinition(start_date="2020-01-01"),
    backfill_policy=BackfillPolicy.single_run(),
)
def events(context: AssetExecutionContext, raw_events):
    output_data = compute_events_from_raw_events(raw_events)
    return output_data
```

Reply:
Agreed. I added this in but am getting errors complaining about this. Could you take a look at the updated example (`single_run_backfill_io_manager.py`) and corresponding test if you have a sec?

Reply:
I think the issue is compute_events_from_raw_events doesnt exist, not sure where this function came from?